### ISSUE TITLE
throw error on not a contact error on sendtext

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2641]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.34.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.33.8...4.34.0) (2022-04-10)

- Release 4.34.0 [`5e1cace`](https://github.com/open-wa/wa-automate-nodejs/commit/5e1caced380fc5aff15a7cd81b5d8ec7b7876ee4)
- 🥅 respect `Config.OnError` on `sendText` errors #2641 [`7aa16e5`](https://github.com/open-wa/wa-automate-nodejs/commit/7aa16e5ecced068f60bc8c94a1aa95fea107793d)
- 🚑 fix: fix missing data for sticker api #2640 [`0ef25bf`](https://github.com/open-wa/wa-automate-nodejs/commit/0ef25bf70c329713ab0176e1acb8a169b2581aa2)
- updated types-only package [`1492e73`](https://github.com/open-wa/wa-automate-nodejs/commit/1492e7374876a1e734fc37e910ff2b3a7a9b9f81)
